plasmapy/transport/classical/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+The `~plasmapy.formulary` subpackage contains commonly used formulae
+from plasma science.
+"""
+# __all__ will be auto populated below
+__all__ = []
+
+from .base import *
+from .epperlein_haines import *
+
+
+# auto populate __all__
+for obj_name in list(globals()):
+    if not (obj_name.startswith("__") or obj_name.endswith("__")):
+        __all__.append(obj_name)
+__all__.sort()
```

Comment:
Isn't the result of this what you get if you don't manually define `__all__` at all and just do the star imports?

Reply:
I've no idea, I just copied this from the formulary module!